src/lib.ts

```diff
@@ -648,4 +649,75 @@ export const generalStoreXpEffect: Effect = {
   Muscle: $effect`Go Get 'Em, Tiger!`,
   Mysticality: $effect`Glittering Eyelashes`,
   Moxie: $effect`Butt-Rock Hair`,
-}[mainStatStr];
\ No newline at end of file
+}[mainStatStr];
+
+export function goVote(): void {
+
+const initPriority: Map<string, number> = new Map([
+  ["Meat Drop: +30", 1,],
+  ["Item Drop: +15", 4],
+  ["Booze Drop: +30", 4],
+  ["Adventures: +1", 3],
+  ["Familiar Experience: +2", 2],
+  ["Monster Level: +10", 3],
+  [`${mainStat} Percent: +25`, 3],
+  [`Experience (${mainStat}): +4`, 2],
+  ["Meat Drop: -30", -4],
+  ["Item Drop: -15", -4],
+  ["Familiar Experience: -2", -4],
+  ["Hot Resistance: +3", 2],
+  ["Weapon Damage Percent: +100", 5],
+  ["Spell Damage Percent: +20", 3]
+]);
+
+const voteLocalPriorityArr = [1, 2, 3, 4].map((index) => ({
+  urlString: index - 1,
+  value:
+    initPriority.get(get(`_voteLocal${index}`)) ??
+    (get(`_voteLocal${index}`).includes("-") ? -1 : 1),
+}));
+
+const init = maxBy(voteLocalPriorityArr, "value").urlString;
+
+
+//Dear Panto,
```

Comment:
I think we can have a hidden setting `instant_optimalVoters` (i.e. don't list it in `savedresources`, similar to `targetBaseMyst`) that defaults to `false` (rotating suboptimally), but can be set to `true` to always choose the mutant